test/core/binary.wast

```diff
@@ -6,6 +6,9 @@
 (assert_malformed (module binary "") "unexpected end")
 (assert_malformed (module binary "\01") "unexpected end")
 (assert_malformed (module binary "\00as") "unexpected end")
+(assert_malformed (module binary "\00asm") "unexpected end")
+(assert_malformed (module binary "\00asm\01") "unexpected end")
+(assert_malformed (module binary "\00asm\01\00\00") "unexpected end")
```

Comment:
I'd prefer to keep this where they were before. That was grouping all tests concerned with the version.

Reply:
Sure, done. :)